aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -132,6 +133,40 @@ module aptos_framework::aptos_account {
         coin::deposit<CoinType>(to, coins)
     }
 
+    /// Batch version of transfer_fungible_assets.
+    public entry fun batch_transfer_fungible_assets(
+        from: &signer,
+        recipients: vector<address>,
+        metadata: Object<Metadata>,
+        amounts: vector<u64>
+    ) {
+        let recipients_len = vector::length(&recipients);
+        assert!(
+            recipients_len == vector::length(&amounts),
+            error::invalid_argument(EMISMATCHING_RECIPIENTS_AND_AMOUNTS_LENGTH),
+        );
+
+        vector::enumerate_ref(&recipients, |i, to| {
+            let amount = *vector::borrow(&amounts, i);
+            transfer_fungible_assets(from, *to, metadata, amount);
+        });
```

Comment:
Similarly, let's keep metadata first rather than splitting the amounts and senders
```suggestion
    public entry fun batch_transfer_fungible_assets(
        from: &signer,
        metadata: Object<Metadata>,
        recipients: vector<address>,
        amounts: vector<u64>
    ) {
        let recipients_len = vector::length(&recipients);
        assert!(
            recipients_len == vector::length(&amounts),
            error::invalid_argument(EMISMATCHING_RECIPIENTS_AND_AMOUNTS_LENGTH),
        );

        vector::enumerate_ref(&recipients, |i, to| {
            let amount = *vector::borrow(&amounts, i);
            transfer_fungible_assets(from, *to, metadata, amount);
        });
```